app/forms/mappings/Formatters.scala

```diff
@@ -78,6 +78,51 @@ trait Formatters {
         baseFormatter.unbind(key, value.toString)
     }
 
+  private[mappings] def numericFormatter(isCurrency: Boolean = false,
```

Comment:
this numeric formatter used for numeric validation is part of the numeric input type which is not used

Reply:
I think this will be used in the currency entry fields